packaging/debs/Debian/debian/control

```diff
@@ -7,11 +7,12 @@ Uploaders: Alvaro Videla <alvaro@rabbitmq.com>,
  Jean-Sébastien Pédron <jean-sebastien@rabbitmq.com>,
  Giuseppe Privitera <giuseppe@rabbitmq.com>
 Build-Depends: debhelper (>= 9),
+ dh-systemd (>= 1.5),
```

Comment:
@michaelklishin and surely that's the very reason APT provides the ability for package suppliers to upload different packages for different releases.  The problem here is the rabbit team's blind insistence on forcing together some frankenstein "one size fits all" package.